draft-ietf-rats-corim.md

```diff
@@ -1457,19 +1460,24 @@ trustworthiness properties of the subject domain exists.
 ~~~ cddl
 {::include cddl/domain-dependency-triple-record.cddl}
 ~~~
+~~~ cddl
+{::include cddl/domain-type-choice.cddl}
+~~~
 
 #### Domain Membership Triple {#sec-comid-triple-domain-membership}
 
-A Domain Membership triple assigns domain membership to environments.  The
-subject identifies a domain ({{sec-comid-domain-type}}) that has a predicate
-relationship to the object containing one or more environments.  Endorsed
-environments ({{sec-comid-triple-endval}}) membership is conditional upon
-successful matching of Reference Values ({{sec-comid-triple-refval}}) to
-Evidence.
+A Domain Membership triple assigns a domain identifier to member environments. The
+triple subject identifies a domain ({{sec-comid-domain-type}}) that has a membership predicate
+relationship to the object where the object contains one or more members.
+When predicates are domains, it allows Endorsers to construct a hierarchical model
+where a domain can be represented using a collection of well defined domain identifiers.
```

Comment:
Not very clear, I suggest the following rewrite:
```suggestion
A Domain Membership triple links a domain identifier to its member Environments.
The triple’s object lists all the Environments in the domain, and the domain is identified by the Environment in the triple’s subject.
This allows the recursive construction of the topology of a Composite Device, where multiple lower-level domains can be aggregated into higher-level domains.
```

Reply:
Taken all the points from here and incorporated a generic term entity with an example of Composite Device!